snuba/web/rpc/v1/resolvers/R_eap_items/storage_routing/routing_strategies/linear_bytes_scanned_storage_routing.py

```diff
@@ -97,7 +93,7 @@ def _get_bytes_scanned_limit(self) -> int:
         return cast(
             int,
             state.get_int_config(
-                _SAMPLING_IN_STORAGE_PREFIX + "bytes_scanned_per_query_limit",
+                self.config_key() + "_bytes_scanned_per_query_limit",
```

Comment:
FYI: if this is set in runtime config somewhere, you should make sure this key is updated